scripts/validate

```diff
@@ -15,5 +15,5 @@ if [[ -z "$(command -v golangci-lint)" ]]; then
 fi
 
 echo "Running: golangci-lint run"
-golangci-lint run
+golangci-lint run --timeout 10m
```

Comment:
Hi @ibrokethecloud,
Can you help to explain why we need to introduce this timeout? Thanks.

Reply:
the default timeout is 1m, and I noticed random failures due to golangci-lint timeout. This is likely due to the lack of resources on the runners.